src/api/app/controllers/webui/project_controller.rb

```diff
@@ -24,6 +24,8 @@ class Webui::ProjectController < Webui::WebuiController
   before_filter :load_releasetargets, :only => [ :show, :incident_request_dialog, :release_repository_dialog ]
   prepend_before_filter :lockout_spiders, :only => [:requests, :rebuild_time, :buildresults, :maintenance_incidents]
 
+  after_action :verify_authorized, :only => :save_new
```

Comment:
@bgeuken I just stick to the official documentation...
https://github.com/elabs/pundit#ensuring-policies-are-used
This will only raise an exception if there is not authorization in this action at all! To prevent you from forget authorization...